packages/x-components/package.json

```diff
@@ -53,14 +53,17 @@
     "prepublishOnly": "npm run build",
     "test:unit": "jest --silent && npm run test:unit-cypress",
     "test:unit-jest": "jest",
-    "test:unit-cypress": "cypress run-ct",
+    "test:unit-cypress": "cypress run --component",
     "test:unit-coverage": "jest --coverage",
     "test:unit-watch": "jest --watch",
-    "test:e2e": "vue-cli-service test:e2e",
+    "test:e2e": "vue-cli-service test:e2e --e2e --browser chrome",
+    "test:e2e:firefox": "vue-cli-service test:e2e --e2e --browser firefox",
     "test:e2e:ci": "vue-cli-service test:e2e --headless --browser chrome",
     "test": "npm run test:unit && npm run test:e2e:ci",
-    "cypress:open": "cypress open",
-    "cypress:open:component": "cypress open-ct"
+    "cypress:open": "cypress open --e2e --browser chrome",
+    "cypress:open:firefox": "cypress open--e2e --browser firefox",
```

Comment:
**nitpick**: relying on the implicit default behaviour of the `cypress open` command

```suggestion
    "cypress:open:firefox": "cypress open --browser firefox",
```

**typo(blocking)**: there's a missing whitespace between open and `--e2e` parameter

```suggestion
    "cypress:open:firefox": "cypress open --e2e --browser firefox",
```